pkg/cri/cri.go

```diff
@@ -97,8 +99,12 @@ func initCRIService(ic *plugin.InitContext) (interface{}, error) {
 	if err != nil {
 		return nil, errors.Wrap(err, "failed to create containerd client")
 	}
+	pf, err := getPortForwardService(ic)
+	if err != nil {
+		return nil, errors.Wrap(err, "failed to get port forward service")
```

Comment:
should we ignore if there is no port forward service to fall back to the prior behavior..  

@dmcgowan @crosbymichael we should do a call I think with Ian to go over this

Reply:
I sent an invite for Wed. Hopefully that works.